### PR TITLE
Vertical Navigation: add tertiary support,  make usability improvements

### PR DESCRIPTION
**Pull request by jeff-phillips-18**
Improvements:
- Change the delay for the initial hover over
- Primary and secondary menus should always be the same width
- Non-persistent menu behavior should be an option
- Change the size of the main navigation icons (from 14px to 20px)
- Support tooltips for the icons in the primary navigation bar

*\* Note: This commit also removes the Alternate Layouts from the tests pages as it is recommended that
applications use the Vertical Navigation components.

Migrate persistent secondary navigation:

Navigation Component (with nav-pf-vertical selector):
- change “nav-pf-vertical-with-secondary-nav” selector to “nav-pf-persistent-secondary nav-pf-vertical-with-sub-menus”
- to use collapsible secondary menus add selector “nav-pf-vertical-collapsible-menus”
- to hide primary menu bar icons add “hidden-icons-pf”

For primary menu items w/ secondary menus:
- change “persistent-secondary” selector to “secondary-nav-item-pf”

For secondary menus:
- change “nav-pf-persistent-secondary” selector to “nav-pf-secondary-nav”
- change “persistent-secondary-header" selector to “nav-item-pf-header"

Container component:
- change “container-pf-nav-pf-vertical-with-secondary” selector to container-pf-nav-pf-vertical-with-sub-menus”

### CODE REVIEW

File: less/vertical-nav.less [hunk 1]
```diff
@@ -34,191 +18,216 @@
   .ie9.layout-pf-fixed & {
     box-sizing: content-box; // IE9 incorrectly sizes the width if using padding-box
   }
-  &.collapsed:not(.nav-pf-vertical-with-secondary-nav) {
-    display: none;
+  .layout-pf-fixed-with-footer & {
```
Comment (srambach) [inline less/vertical-nav.less:21]: Can you add some comments to clarify what's going on throughout the file? Especially with & selectors, it's really easy to get lost quickly, and it's a huge file with tons of nesting.

File: less/vertical-nav.less [hunk 2]
```diff
@@ -505,24 +703,232 @@
...
+  }
+  .tertiary-collapse-toggle-pf {
+    display: none;
+    font-family: @icon-font-name-fa;
+    font-size: inherit;
+    opacity: 0;
+    pointer-events: none;
+    -webkit-font-smoothing: antialiased;
+    &:before {
+      content: '\f190';
```
Comment (srambach) [inline less/vertical-nav.less:880]: It's helpful to have a comment saying what character this actually is when using the codes

File: less/vertical-nav.less [hunk 3]
```diff
@@ -34,191 +18,216 @@
...
-        }
-        &.collapsed {
-          width: @nav-pf-vertical-collapsed-width;
-          &.hover-secondary-nav-pf, &.secondary-visible-pf {
-            width: (@nav-pf-vertical-collapsed-width + @nav-pf-vertical-secondary-width);
-          }
-          &.collapsed-secondary-nav-pf {
-            width: @nav-pf-vertical-secondary-width;
-          }
+      &tertiary-nav-item:hover {
```
Comment (srambach) [inline less/vertical-nav.less:199]: Is this a typo or intentional concatenation?

File: less/vertical-nav.less [hunk 4]
```diff
@@ -34,191 +18,216 @@
...
+    &.show-mobile-secondary {
+      width: @nav-pf-vertical-width;
+    }
+    &.show-mobile-tertiary {
       width: @nav-pf-vertical-width;
-      &.collapsed {
-        width: @nav-pf-vertical-collapsed-width;
+    }
+
+    .mobile-nav-item-pf, .mobile-secondary-item-pf {
```
Comment (srambach) [inline less/vertical-nav.less:183]: Could you put all the multiple selectors on separate lines for better readability?

File: less/vertical-nav.less [hunk 5]
```diff
@@ -1,29 +1,13 @@
 //
 //  Vertical navigation
 // --------------------------------------------------
```
Comment (srambach) [inline less/vertical-nav.less:3]: Is there any possible way we could simplify this entire pattern to get rid of so much nesting? I just used the two-level vertical navigation for something and just to remove the caret, I had to write a selector 6 levels deep to overcome the specificity. It's probably beyond the scope of this PR but I wanted to bring it up.

**Comment by jeff-phillips-18** [comment]
Ah yes, I meant to re-go over this file prior to putting up the PR. Got a bit ahead of myself.

**Pull request closed without merging**